arduino/libraries/librariesmanager/install.go

```diff
@@ -133,6 +133,11 @@ func (lm *LibrariesManager) InstallZipLib(ctx context.Context, archivePath strin
 
 	extractionPath := paths[0]
 	libraryName := extractionPath.Base()
+
+	if err := validateLibrary(libraryName, extractionPath); err != nil {
+		return err
```

Comment:
Probably a stupid question: why the extraction path is not cleared as done [here](https://github.com/arduino/arduino-cli/pull/1234/files#diff-7201cbc75bf18b11ac71b3a4a12caae78081c2d6be4e164db68db73ba10a0ea6R229)?

Reply:
There's a `defer tmpDir.RemoveAll()` at line `108`, we don't do the same when installing from git url since we clone directly in the installation folder in that case.